libs/sistema/theme/tokens/_buttons.tokens.scss

```diff
@@ -5,22 +5,44 @@
 //=====================================
 // BUTTONS
 //=====================================
+$border-radius-button: $border-radius-default;
+
 // Borders
-$border-radius-button: rhythm(0.5);
+$border-button-primary-basic: 1px solid $color-primary-light;
+$border-button-primary-basic-hover: 1px solid $color-primary-light;
+$border-button-primary-basic-active: 1px solid $color-primary-light;
+
+$border-button-secondary-basic: 1px solid $color-neutral-light;
+$border-button-secondary-basic-hover: 1px solid $color-neutral-light;
+$border-button-secondary-basic-active: 1px solid $color-neutral-light;
+
+$border-button-inverted-basic: 1px solid $color-neutral-light;
+$border-button-inverted-basic-hover: 1px solid $color-neutral-light;
+$border-button-inverted-basic-active: 1px solid $color-neutral-light;
+
+$border-button-destructive-basic: 1px solid $color-secondary-strong;
+$border-button-destructive-basic-hover: 1px solid $color-secondary-strong;
+$border-button-destructive-basic-active: 1px solid $color-secondary-strong;
+
+$border-button-basic-disabled: 1px solid $color-neutral-lighter;
+$border-button-inverted-basic-disabled: 1px solid $color-neutral-inverted-lighter;
 
 // Sizes
-$padding-button-medium: rhythm(0.5) rhythm(2);
-$padding-button-large: rhythm(1) rhythm(2);
+$padding-sides-button-large: rhythm(2);
 
-$padding-button-icon-text-medium: rhythm(0.5) rhythm(2) rhythm(0.5) rhythm(1.5);
-$padding-button-icon-text-large: rhythm(1) rhythm(2) rhythm(1) rhythm(1.5);
+$padding-left-button-icon-text-large: rhythm(1.5);
+$padding-right-button-icon-text-large: rhythm(2);
 
 $padding-button-icon-medium: rhythm(1);
 $padding-button-icon-large: rhythm(1);
 
 $font-size-button-medium: font-size(s);
 $font-size-button-large: font-size(s);
 
+$height-button-small: rhythm(4);
```

Comment:
A small thing: the text in `pa-buttons` is not vertically aligned like before, is this expected ?

Reply:
@operramon I'm not able to reproduce, did you update you're Pastanaga version locally before testing it?